docs/source/guides/saving_and_loading_dataframes.ipynb

```diff
@@ -0,0 +1,275 @@
+    "            \"sep\": \",\",\n",
+    "            \"encoding\": \"utf-8\",\n",
+    "            \"engine\": \"python\",\n",
+    "            \"index\": false\n",
+    "        }\n",
+    "    },\n",
+    "    \"table_metadata\": {}\n",
+    "}\n",
+    "```\n",
+    "\n",
+    "## Loading a Woodwork DataFrame\n",
+    "\n",
+    "After saving a Woodwork DataFrame, you can load the DataFrame and typing information by using `woodwork.deserialize.read_woodwork_table`. This function will use the stored typing information in the specified directory to recreate the Woodwork DataFrame."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "oriental-baking",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from woodwork.deserialize import read_woodwork_table\n",
+    "df = read_woodwork_table('retail')\n",
+    "df.ww.schema"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dependent-liberty",
+   "metadata": {},
+   "source": [
+    "### Loading the DataFrame and typing information separately\n",
+    "\n",
+    "You can also load the Woodwork DataFrame and typing information separately by using `woodwork.read_file`. This function uses the `content_type` parameter to determine the file format. If `content_type` is not specified, the function will try to infer the file format from the file extension. The typing information such as the index, time index, logical types, and semantics tags are optional parameters. This approach is helpful if you want to save and load the typing information outside the specified directory or read a data file directly into a Woodwork DataFrame. To illustrate, I will write a pandas DataFrame to a single file in different formats, then load it into a Woodwork DataFrame using this typing information."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "automotive-married",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "typing_information = {\n",
+    "    'index': 'order_product_id',\n",
+    "    'time_index': 'order_date',\n",
+    "    'logical_types': {\n",
+    "        'order_product_id': 'Categorical',\n",
+    "        'order_id': 'Categorical',\n",
+    "        'product_id': 'Categorical',\n",
+    "        'description': 'NaturalLanguage',\n",
+    "        'quantity': 'Integer',\n",
+    "        'order_date': 'Datetime',\n",
+    "        'unit_price': 'Double',\n",
+    "        'customer_name': 'Categorical',\n",
+    "        'country': 'Categorical',\n",
+    "        'total': 'Double',\n",
+    "        'cancelled': 'Boolean',\n",
+    "    },\n",
+    "    'semantic_tags': {\n",
+    "        'order_id': {'category'},\n",
+    "        'product_id': {'category'},\n",
+    "        'quantity': {'numeric'},\n",
+    "        'unit_price': {'numeric'},\n",
+    "        'customer_name': {'category'},\n",
+    "        'country': {'category'},\n",
+    "        'total': {'numeric'},\n",
+    "    }\n",
```

Comment:
As part of this example, should we actualy load the JSON in from file separately instead of / in addition to defining it?

Reply:
The example was updated to load in the JSON, but it seems like the JSON schema doesn't directly map to the typing information parameters in `read_file`. How do we want to convert `column_typing_info` to logical types and semantic tags?

Reply:
Yeah that's kind of messy.  @thehomebrewnerd @tamargrey do you envision `read_file` being used without the JSON?  What scenarios do we want to document?

Reply:
**read_file** can be used with/without the JSON. It can also be used if your typing information is stored in a database and you want to pass it in with `read_file`

Reply:
@gsheni that was poorly worded, I mean if `read_file` was meant to be used both with and without the JSON, since you'd need to do some preprocessing on the JSON info first before feeding it to `read_file`

Reply:
**read_file** isn't meant to be used with the JSON generated from `to_disk`. It is meant to be used where the user wants to store typing information in their own way.

Reply:
Then my intial suggestion was a poor one and we shouldn't use the JSON here

Reply:
But perhaps we should put the sentence explaining why one might want to do this as the second sentence in the paragraph, to communicate why you might want to do this before explaining the specifics of how

Reply:
Not sure if this helps, but I tend to think of `read_file` as an analog to `pandas.read_csv`. It's just an easy helper function to take data stored on disk in some format and read it into a Woodwork initialized dataframe. You don't need to specify any logical types or semantic tags if you don't want to, but if you want more control over how things get initialized you can pass those values as well. It just allows you to combine the read and init functions in a single call instead of having to call the library read function and then initialize Woodwork.

The `to_disk` method coupled with the `read_woodwork_table` function are what you would use to serialize and deserialize a Woodwork dataframe if you want the deserialized table to match exactly the original dataframe, including all the typing info.

Reply:
Which I see you already did, I was looking at an old build

Reply:
I reverted the change and updated the comments to help clarify when we would want to use this approach.

Reply:
looks good 👍 

what do you think of adding an example of the simple use Nate talked about -- just calling `read_file`, no typing info -- at the beginning to demonstrate a quick way to get a woodwork DF?

Reply:
Sure I'll add that